cmd/helm/dependency_update_test.go

```diff
@@ -182,7 +182,7 @@ func TestDependencyUpdateCmd_DontDeleteOldChartsOnError(t *testing.T) {
 func createTestingMetadata(name, baseURL string) *chart.Chart {
 	return &chart.Chart{
 		Metadata: &chart.Metadata{
-			APIVersion: chart.APIVersionV1,
+			APIVersion: chart.APIVersionV2,
```

Comment:
Can you make this change also in the following places:
- https://github.com/helm/helm/blob/master/pkg/downloader/manager_test.go#L214
- https://github.com/helm/helm/blob/master/pkg/downloader/manager_test.go#L288